### PR TITLE
GitHub

### PR DESCRIPTION
**Pull request by gidden**
updates the github section based on @SteveGoldstein's comments

### CODE REVIEW

**Comment by gidden** [comment]
this supercedes #170 and will close it automatically upon merge

File: version-control/git/github/Readme.md [hunk 1]
```diff
@@ -44,17 +44,26 @@ you want an
...
-A key step to interacting with an online repository that you have forked
-is adding the original as a remote repository. By adding the remote
-repository, you inform git of a new option for fetching updates and
-pushing commits.
+From [Github](https://github.com/octocat/Spoon-Knife):
 
-The **git remote** command allows you to add, name, rename, list, and
-delete repositories such as the original one **upstream** from your
-fork, others that may be **parallel** to your fork, and so on.
+> Creating a fork is producing a personal copy of someone else's project. A fork
```
Comment (SteveGoldstein) [inline version-control/git/github/Readme.md:49]: italicize "fork"

File: version-control/git/github/Readme.md [hunk 2]
```diff
@@ -44,17 +44,26 @@ you want an
...
+> acts as a bridge between the original repository and your personal copy. You can
+> submit Pull Requests to help make other people's projects better by offering
```
Comment (SteveGoldstein) [inline version-control/git/github/Readme.md:51]: italicize "Pull Requests"

**Comment by gidden** [comment]
Hi @SteveGoldstein, @lmichael107 already pulled this. Can you provide a PR with your suggested edits?

**Pull request merged**